$MirOS: ports/print/cups/patches/patch-config-scripts_cups-directories_m4,v 1.3 2006/11/08 18:25:44 bsiegert Exp $
--- config-scripts/cups-directories.m4.orig	Mon Mar 12 21:48:50 2007
+++ config-scripts/cups-directories.m4	Thu Mar 15 19:14:44 2007
@@ -148,7 +148,7 @@ if test x$rcdir = x; then
 			fi
 			;;
 
-		FreeBSD* | OpenBSD* | MirBsD* | ekkoBSD*)
+		FreeBSD* | OpenBSD* | MirBSD* | ekkoBSD*)
 			# FreeBSD and OpenBSD
 			;;
 
